Examples/Scripts/PreFreeSurferPipelineBatchNHP.sh

```diff
@@ -0,0 +1,335 @@
+    echo ""
+    echo "Optional Options:"
+    echo "  --BrainExtract: INVIVO (default) or EXVIVO"
+    echo "  --UnwarpDir: Unwarp direction (default: z)"
+    echo "  --MION: Use MION (default: 0)"
+    echo ""
+    exit 1;
+}
+
+# ==== User-editable section ====
+# Edit these variables before running
+StudyFolder="${HOME}/projects/Pipelines_ExampleData"
+Subjlist="100307 100610"
+SPECIES="Human"
+RunMode="Default"
+BrainExtract="INVIVO"
+StrucRes="0.8"
+
+# Parse command line arguments
+get_batch_options() {
+    local arguments=("$@")
+    
+    local index=0
+    local numArgs=${#arguments[@]}
+    local argument
+    
+    while [ ${index} -lt ${numArgs} ]; do
+        argument=${arguments[index]}
+        
+        case ${argument} in
+            --StudyFolder=*)
+                StudyFolder=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --Subject=*)
+                Subjlist=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --Species=*)
+                SPECIES=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --RunMode=*)
+                RunMode=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --StrucRes=*)
+                StrucRes=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --BrainExtract=*)
+                BrainExtract=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --UnwarpDir=*)
+                UnwarpDir=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            --MION=*)
+                MION=${argument#*=}
+                index=$(( index + 1 ))
+                ;;
+            *)
+                echo ""
+                echo "ERROR: Unrecognized Option: ${argument}"
+                echo ""
+                Usage
+                ;;
+        esac
+    done
+}
+
+# Parse arguments
+get_batch_options "$@"
+
+# Check required parameters
+if [ -z "$StudyFolder" ] || [ -z "$Subjlist" ] || [ -z "$SPECIES" ] || [ -z "$RunMode" ] || [ -z "$StrucRes" ]; then
+    echo "ERROR: Missing required parameters"
+    Usage
+fi
+
+if [ -z "$UnwarpDir" ]; then
+    if [ "$SPECIES" != "Human" ]; then
+        UnwarpDir="z-"
+    else
+        UnwarpDir="z"
+    fi
+fi
+
+if [ -z "$MION" ]; then
+    MION="0"
+fi
+
+echo "$(basename $0) $@"
+
+if [ -z ${EnvironmentScript} ] ; then
+    EnvironmentScript="$HCPPIPEDIR/Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+source $EnvironmentScript
+
+# species specific config
+# If StrucRes is not provided, SetUpSPECIES.sh will use species-specific default
+#if you used MION, change the fix training data by uncommenting this line and providing the path to the training file if necessary.
+#TrainingData=NHPHCP_Macaque.USPIO
+source "$HCPPIPEDIR"/Examples/Scripts/SetUpSPECIES.sh "$SPECIES" "$StrucRes" "$UnwarpDir"
```

Comment:
Make commented out variable names for the user to overide defaults from Set species.